src/EventStore.Client/EventStoreClientOperationOptions.cs

```diff
@@ -11,6 +11,7 @@ public class EventStoreClientOperationOptions {
 		/// <summary>
 		/// An optional <see cref="TimeSpan"/> to use for gRPC deadlines.
 		/// </summary>
+		[Obsolete]
```

Comment:
Probably remove `TimeoutAfter` altogether since it doesn't do anything any more. Obsolete is nice for when something is deprecated but still works (imho)